app/views/govuk_publishing_components/components/docs/radio.yml

```diff
@@ -168,6 +168,19 @@ examples:
         text: "Green"
       - value: "blue"
         text: "Blue"
+  with_custom_heading_size:
```

Comment:
What's changed here?

Reply:
Defined a new example for the custom `heading_size` set to "s"